code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -17,24 +17,30 @@
 	layer = GAS_SCRUBBER_LAYER
 	hide = TRUE
 	shift_underlay_only = FALSE
+	pipe_state = "uvent"
+	vent_movement = VENTCRAWL_ALLOWED | VENTCRAWL_CAN_SEE | VENTCRAWL_ENTRANCE_ALLOWED
 
+	///Direction of pumping the gas (RELEASING or SIPHONING)
 	var/pump_direction = RELEASING
-
+	///Flag to indicate what pressure check to implement (EXT_BOUND, INT_BOUND, NO_BOUND)
```

Comment:
This is a bit like, ahhh dry? pump_direction is actually a lot better, more clear

Reply:
dunno if i made it better